src/Components/Search/Suggestions/SuggestionItem.tsx

```diff
@@ -110,15 +114,35 @@ const DefaultSuggestion: React.FC<SuggestionItemProps> = ({
   )
 
   return (
-    <>
-      <Text variant="sm" overflowEllipsis>
-        {partTags}
-      </Text>
-
-      <Text color="black60" variant="xs" overflowEllipsis>
-        {label}
-      </Text>
-    </>
+    <Flex>
+      <Box width={50} height={50} mr={1}>
+        {imageUrl ? (
+          <Image
+            lazyLoad
+            width={50}
+            height={50}
+            src={imageUrl}
+            borderRadius={label === "Artist" ? 50 : 0}
+          />
+        ) : (
+          <Box
+            width={50}
+            height={50}
+            bg="black5"
+            borderRadius={label === "Artist" ? 50 : 0}
+          />
+        )}
+      </Box>
+      <Flex flexDirection="column" mr={1} flex={1} overflow="hidden">
```

Comment:
**question**: do we need to use `mr={1}` and `overflow="hidden"` here?

Reply:
we also need to center the text vertically

<img width="927" alt="Artsy — Discover, Buy, and Sell Fine Art 2023-03-17 17-47-07" src="https://user-images.githubusercontent.com/3513494/225938588-af4c4e75-dddc-4e20-aaa5-77d43f50a305.png">

Reply:
> do we need to use `mr={1}` and `overflow="hidden"` here?

We need `mr={1}` or `mx={1}` to give a space between the image and label. `<Separator mx={1} />` is not that helpful. 
Also we need `overflow="hidden"` to avoid the issue in ss below 

<img width="906" alt="image" src="https://user-images.githubusercontent.com/17421923/226287171-5af8ab01-9282-4125-8e64-dabe455c0c50.png">

Reply:
> we need overflow="hidden" to avoid the issue in ss below

Hmm, strange. this behavior should not be, since `overflowEllipsis` is used for text

<img width="927" alt="Collector Profile | Artsy 2023-03-20 15-33-35" src="https://user-images.githubusercontent.com/3513494/226340011-9a424cb8-444a-4534-93df-cddcaa9a65fe.png">

Reply:
we don't need to specify the right padding for the text ([mx={1}](https://github.com/artsy/force/blob/b042c3249537c90bd9df52bc1399b054b9f594be/src/Components/Search/Suggestions/SuggestionItem.tsx#L119) here), since [the parent container](https://github.com/artsy/force/blob/b042c3249537c90bd9df52bc1399b054b9f594be/src/Components/Search/Suggestions/SuggestionItem.tsx#L93) already has `px: 2`. 

```javascript
<SuggestionItemPreview imageUrl={imageUrl} label={label} />
{/* replaced `mx={1}` on `ml={1}` */}
<Flex flexDirection="column" ml={1} flex={1} overflow="hidden">...</Flex>
```

or

```javascript
<SuggestionItemPreview imageUrl={imageUrl} label={label} />

<Spacer x={1} />

<Flex flexDirection="column" flex={1} overflow="hidden">...</Flex>
```